lib/googleHelpers.js

```diff
@@ -0,0 +1,60 @@
+const { OAuth2Client } = require('google-auth-library')
```

Comment:
If most of this code came from documentation or a website somewhere, could you please add a link in the comments somewhere in case anyone wants to look it up again in the future?

Reply:
Sure! I'll dig around for some specific documentation on the use of `OAuth2Client`... most of the code here has been a conglomeration of Google documentation, and various guides, though, so it'll be hard to find something that matches how I've gone about using Google here.

Reply:
In that case, don't worry about it too much unless you remember something that might be particularly useful. Thanks!